cms/templates/studio_xblock_wrapper.html

```diff
@@ -158,7 +158,15 @@
 
 % if show_preview:
     % if is_root or not xblock_url:
-        <article class="xblock-render">
+        % if not is_root:
```

Comment:
If that condition is not there, then the lang attribute will get added higher up in the tree then we might want (i.e. will include the edit buttons at the top). We just wanted the attribute to be for the words of the course content itself.

Will make the simplification change.